meinberlin/assets/scss/components/_dashboard_nav.scss

```diff
@@ -114,6 +114,10 @@ input:checked ~ .dashboard-nav__menu-content {
     border-top-right-radius: 0;
     border-bottom-left-radius: 0;
     max-width: 48px;
+
+    i {
+        font-size: 1rem;
+    }
 }
```

Comment:
this might stop the iphone scaling issue but i'm not sure

Reply:
This is merging and trying again, right? Did you also remove the bit that didn't work?

Reply:
Yes, you did!